doc/ref_arch/openstack/chapters/README.rst

```diff
@@ -0,0 +1,60 @@
+OpenStack based Reference Architecture
+======================================
+
+This is OpenStack based Reference Architecture (RA-1)
+
+Release Information
+-------------------
+
+**Bundle: 6**
+
+**Version: 0**
+
+**Release Date: 4th January 2022**
+
+Bundle/Version History
+----------------------
+
+============== =================== ================
+Bundle.Version Date                Note
+============== =================== ================
+1.0            20th September 2019 Botrange Release
+2.0-alpha      10th January 2020   Snezka Release
+2.0            15th May 2020       Baldy Release
+3.0            25th Sep 2020       Baraque Release
+4.0            29th Jan 2021       Elbrus Release
+5.0            1st July 2021       Kali Release
+6.0            4th January 2022    Lakelse Release
+============== =================== ================
+
+Overall Status
+--------------
+
+========== ========
+Chapter    Status
+========== ========
+Chapter 01 Complete
+Chapter 02 Complete
+Chapter 03 Complete
+Chapter 04 Complete
+Chapter 05 Complete
+Chapter 06 Complete
+Chapter 07 Complete
+Chapter 08 Complete
+========== ========
+
+Table of Contents
+-----------------
+

```

Comment:
@karinesevilla should these all be w/o md so that they link to rst files?

Reply:
I think a `toctree` with all the chapters would be the best solution here.

Reply:
A link index.rst to README.rst makes it work smoothly

Reply:
```
.. toctree::
   :glob:
   :maxdepth: 1

   chapters/*
```